engine/src/test/java/io/zeebe/engine/processing/streamprocessor/ReplayStateRandomizedPropertyTest.java

```diff
@@ -162,8 +169,8 @@ private void waitForProcessingToStop() {
   @Parameters(name = "{0}")
   public static Collection<TestDataRecord> getTestRecords() {
     // use the following code to rerun a specific test case:
-    //    final var workflowSeed = 8354930473519021610L;
-    //    final var executionPathSeed = 4913527714508155594L;
+    //    final var workflowSeed = 4151337193744273092L;
+    //    final var executionPathSeed = 8078198989481167852L;
```

Comment:
I'm not familiar with these tests yet - is this a convention? When we have a new failing case we update the seeds?

Reply:
No, it was not required to change the seed :laughing: